pydoc/developer/USER_BUGS.md

```diff
@@ -0,0 +1,39 @@
+.. _user_bugs:
```

Comment:
Again, this label won't work natively in markdonw. Update to 
<pre><code>
```{eval-rst}  
.. _user_bugs:
```
</code></pre>

or remove since it is not used.